scripts/pkg/windows/nri-amd64-installer/Product.wxs

```diff
@@ -1,98 +1,97 @@
-                            <Directory Id="CustomPluginsBinFolderIntegration" Name="bin"/>
-                        </Directory>
-                    </Directory>
-                </Directory>
-            </Directory>
-        </Directory>
-    </Fragment>
-
-    <Fragment>
-        <DirectoryRef Id="CustomPluginsBinFolder">
-            <Component Id="CMP_V1_CUSTOM_PLUGINS_BIN" Guid="2244c368-dd8b-404a-bdc0-67c525d7422f" KeyPath="yes">
-                <CreateFolder/>
-            </Component>
-        </DirectoryRef>
-        <DirectoryRef Id="CustomPluginsBinFolderIntegration">
-            <Component Id="CMP_V1_CUSTOM_PLUGINS_BIN_INTEGRATION" Guid="6eac0d38-fcc1-450b-8072-9573092bbfe1" KeyPath="yes">
-                <CreateFolder/>
-            </Component>
-        </DirectoryRef>
-        <DirectoryRef Id="PluginConfigsFolder">
-            <Component Id="CMP_V1_PLUGIN_CONFIGS" Guid="803C978C-B2A1-47C7-BB17-A2DE9E6D3145" KeyPath="yes">
-                <CreateFolder/>
-            </Component>
-        </DirectoryRef>
-    </Fragment>
-
-    <Fragment>
-        <ComponentGroup Id="PrometheusPluginsBinComponent" Directory="CustomPluginsBinFolder">
-            <Component Id="CMP_NRI_EXPORTER_EXE" Guid="{exporterBinGUID}" Win64="yes">
-                <File Id="FILE_NRI_EXPORTER_EXE"
-                      Source="$(var.BinariesPath){IntegrationExe}"
-                      KeyPath="yes"/>
-            </Component>
-            <Component Id="CMP_NRI_LICENSE" Guid="{licenseGUID}" Win64="yes">
-                <File Id="FILE_NRI_LICENSE"
-                      Source="$(var.BinariesPath){IntegrationName}-LICENSE"
-                      KeyPath="yes"/>
-            </Component>
-        </ComponentGroup>
-        <ComponentGroup Id="CustomPluginsBinComponent" Directory="CustomPluginsBinFolderIntegration">
-            <Component Id="CMP_NRI_$(IntegrationName)_EXE" Guid="{nriGUID}" Win64="yes">
-                <File Id="FILE_NRI_$(IntegrationName)_EXE"
-                      Source="$(var.BinariesPath)nri-$(IntegrationName).exe"
-                      KeyPath="yes"/>
-            </Component>
-        <ComponentGroup Id="PluginConfigsComponent" Directory="PluginConfigsFolder">
-            <Component Id="CMP_NRI_EXPORTER_CONFIG" Guid="{configGUID}" Win64="yes">
-                <File Id="CMP_NRI_EXPORTER_CONFIG"
-                      Name="{IntegrationName}-config.yml.sample"
-                      Source="$(var.BinariesPath){IntegrationName}-config.yml.sample"
-                      KeyPath="yes"/>
-            </Component>
-        </ComponentGroup>
-    </Fragment>
-
-</Wix>
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Product Id="*"
+             Name="New Relic Prometheus Exporter Integration, nri-$(var.IntegrationName)"
+             Language="1033"
+             Version="$(var.IntegrationVersion)"

```

Comment:
before the changes in order to use a variable we needed to declare it first. Do you know how can we use them directly now?

Reply:
They are defined here https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/137/files#diff-18a39f21aef81bfebafb618d7701ab708ca8dfe6432217dbb53959ea31076f79R20.